### PR TITLE
fix(.github): adds github workflows for ISSUE TEMPLATES for bug addition, documentation add, feature request and security

### PR DESCRIPTION
**Pull request by ItsRoy69**
Worked on the issue #163 which was assigned to me

bug.yml - Issue Template for bug fixing 
docs.yml - Issue Template for documentation addition
feature_request.yml - Issue Template for requesting a feature
other.yml - Issue Template for other features
SECURITY.md - Issue Template for security issues

### CODE REVIEW

File: .github/workflows/ISSUE_TEMPLATE/docs.yml
Comment (alejandrojnm) [inline .github/workflows/ISSUE_TEMPLATE/docs.yml:1]: I think we don't need this one

**Review by alejandrojnm — COMMENTED** [review]
More than happy to merge, but is it possible to remove the icons?

**Comment by ItsRoy69** [comment]
Sure I'll resolve it

File: .github/workflows/ISSUE_TEMPLATE/docs.yml
Comment (ItsRoy69) [reply .github/workflows/ISSUE_TEMPLATE/docs.yml:1]: Made the necessary changes

**Comment by ItsRoy69** [comment]
Made the necessary changes

**Comment by ItsRoy69** [comment]
@alejandrojnm @alessandroargentieri please can you check the PR ?

**Comment by alejandrojnm** [comment]
Remove all the icons

**Comment by ItsRoy69** [comment]
> Remove all the icons

Made the necessary changes

File: .github/workflows/ISSUE_TEMPLATE/bug.yml
Comment (alejandrojnm) [inline .github/workflows/ISSUE_TEMPLATE/bug.yml:1]: Have icon

File: .github/workflows/ISSUE_TEMPLATE/feature_request.yml
Comment (alejandrojnm) [inline .github/workflows/ISSUE_TEMPLATE/feature_request.yml:1]: Same, icon

File: .github/workflows/ISSUE_TEMPLATE/other.yml
Comment (alejandrojnm) [inline .github/workflows/ISSUE_TEMPLATE/other.yml:1]: Same

**Review by alejandrojnm — COMMENTED** [review]
I leave some comment's

File: .github/workflows/ISSUE_TEMPLATE/other.yml
Comment (ItsRoy69) [reply .github/workflows/ISSUE_TEMPLATE/other.yml:1]: Resolved the icon issues

**Review by alejandrojnm — APPROVED** [review]

**Pull request merged**